sentry_sdk/integrations/graphene.py

```diff
@@ -46,53 +47,55 @@ def _patch_graphql():
 
     def _sentry_patched_graphql_sync(schema, source, *args, **kwargs):
         # type: (GraphQLSchema, Union[str, Source], Any, Any) -> ExecutionResult
-        hub = Hub.current
-        integration = hub.get_integration(GrapheneIntegration)
+        client = sentry_sdk.get_client()
+        integration = client.get_integration(GrapheneIntegration)
         if integration is None:
             return old_graphql_sync(schema, source, *args, **kwargs)
 
-        with hub.configure_scope() as scope:
-            scope.add_event_processor(_event_processor)
+        scope = Scope.get_isolation_scope()
+        scope.generate_propagation_context()
+        scope.add_event_processor(_event_processor)
 
         result = old_graphql_sync(schema, source, *args, **kwargs)
 
         with capture_internal_exceptions():
             for error in result.errors or []:
                 event, hint = event_from_exception(
                     error,
-                    client_options=hub.client.options if hub.client else None,
+                    client_options=client.options,
                     mechanism={
                         "type": integration.identifier,
                         "handled": False,
                     },
                 )
-                hub.capture_event(event, hint=hint)
+                sentry_sdk.capture_event(event, hint=hint)
 
         return result
 
     async def _sentry_patched_graphql_async(schema, source, *args, **kwargs):
         # type: (GraphQLSchema, Union[str, Source], Any, Any) -> ExecutionResult
-        hub = Hub.current
-        integration = hub.get_integration(GrapheneIntegration)
+        client = sentry_sdk.get_client()
+        integration = client.get_integration(GrapheneIntegration)
         if integration is None:
             return await old_graphql_async(schema, source, *args, **kwargs)
 
-        with hub.configure_scope() as scope:
-            scope.add_event_processor(_event_processor)
+        scope = Scope.get_isolation_scope()
+        scope.generate_propagation_context()
+        scope.add_event_processor(_event_processor)
 
         result = await old_graphql_async(schema, source, *args, **kwargs)
 
         with capture_internal_exceptions():
             for error in result.errors or []:
                 event, hint = event_from_exception(
                     error,
-                    client_options=hub.client.options if hub.client else None,
+                    client_options=client.options if client else None,
```

Comment:
```suggestion
                    client_options=client.options,
```

Client is now always there, so no need for that check.